src/main/webapp/app/shared/api/generated/base.ts

```diff
@@ -1,10 +1,10 @@
 /* tslint:disable */
 /* eslint-disable */
 /**
- * OpenAPI definition
- * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ * oncokb-curation API
+ * oncokb-curation API documentation
  *
- * The version of the OpenAPI document: v0
+ * The version of the OpenAPI document: 1.0.0
```

Comment:
I'm not sure what caused the changes. I use the ` yarn updateAPI` with `Now using node v14.21.3 (npm v6.14.18)` yarn ` 1.22.2`